.rubocop.yml

```diff
@@ -28,6 +28,12 @@ Metrics/BlockLength:
   Exclude:
     - 'spec/**/*'
 
+Metrics/MethodLength:
+  CountAsOne:
```

Comment:
Not sure we really need this :)

Reply:
I need this because when I tried to method call with new line for each argument for example
```ruby
Keyword.update(
     keyword.id,
     status: :completed,
     ...
)
```
instead of 
```ruby
Keyword.update(keyword.id, status: :completed, ...)
```

Rubocop will count the first example as one line so the `MethodLength` rules will not raise